mysql-test/suite/rocksdb/t/drop_table-master.opt

```diff
@@ -1,4 +1,3 @@
 --rocksdb_max_subcompactions=1
---rocksdb_info_log_level=info_level
```

Comment:
BTW, we see it frequently in our jenkins farm. If there is anything we can provide to help diagnose, please ask.